src/sc/ContractParam.js

```diff
@@ -64,11 +64,19 @@ class ContractParam {
    * @param {string} format - The format that this value represents. Different formats are parsed differently.
    * @return {ContractParam}
    */
-  static byteArray (value, format) {
+  static byteArray (value, format, decimals = 8) {

```

Comment:
These arguements will not match the method `byteArray`. I would suggest `(value, format, ...args)` so we can be flexible in receiving arguments.